testhelpers/org.eclipse.n4js.tests.helper/src/org/eclipse/n4js/JSLibSingleTestConfigProvider.java

```diff
@@ -85,14 +84,15 @@ public boolean apply(String s) {
 	 *            the classpath-relative location of the to-be-read resource
 	 */
 	private static List<String> getFileLines(final String resourceName) throws IOException {
-		InputSupplier<InputStreamReader> readerSupplier = CharStreams.newReaderSupplier(
-				new InputSupplier<InputStream>() {
-					@Override
-					public InputStream getInput() throws IOException {
-						return Thread.currentThread().getContextClassLoader().getResourceAsStream(resourceName);
-					}
-				}, Charsets.UTF_8);
-		return CharStreams.readLines(readerSupplier);
+		ByteSource byteSource = new ByteSource() {
+			@Override
+			public InputStream openStream() throws IOException {
+				return Thread.currentThread().getContextClassLoader().getResourceAsStream(resourceName);
+			}
+		};
+
+		CharSource charSrc = byteSource.asCharSource(Charsets.UTF_8);
+		return charSrc.readLines();
```

Comment:
if possible, remove this method entirely and use `java.nio.Files#readAllLines()`